lib/manager/bazel/extract.ts

```diff
@@ -4,7 +4,9 @@ import { parse as _parse } from 'url';
 import { logger } from '../../logger';
 import { PackageDependency, PackageFile } from '../common';
 
-function parseUrl(urlString: string) {
+function parseUrl(
+  urlString: string
+): { repo: string; currentValue: string } | null {
```

Comment:
Shouldn't we add a type or interface here?

Reply:
Something like this?

```ts
export interface UrlParsedResult {
  repo: string;
  currentValue: string
}
```